### DISCUSSION TITLE
is OpenSnitch a frontend ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#591] [Q&A] [answered]
Hi everyone,

### Intro
Sorry for this maybe *dumb* question. But I begin my quest in the _GNU/Linux_ world.
and this tool is exactly (what I believe) should have a the majority of the Linux distribution...

### The question
is OpenSnitch a frontend of the Linux Kernel : Netfilter ?
if so I guess all the others frontend (FirewallD, IPtable etc..) should be disabled ?

Thanks.

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
Hi @BobSquarePants ,

OpenSnitch is not a "frontend" of Netfilter, because right now you can't configure system (Netfilter) rules. OpenSnitch uses Netfilter and other technologies to control applications outbound connections, allowing you to create rules for those applications. But the rules are not added to Netfilter. Because of this, firewalld, iptables or nftables are still needed if you want to block incoming connections, forward ports, etc...

For v1.6.0 release (hopefully), you'll be able to configure system rules from the GUI, allowing you to add whatever rule you want to add (like blocking incoming connections while allowing ESTABLISHED ones + published network services, etc..).

**Comment by BobSquarePants** [comment]
Thank you @gustavo-iniguez-goya that look really promising !
How can we help this great project ?
